src/main/scala/org/broadinstitute/hail/Utils.scala

```diff
@@ -596,17 +598,17 @@ class RichPairTraversableOnce[K, V](val t: TraversableOnce[(K, V)]) extends AnyV
   }
 }
 
-class RichPairIterator[K, V](val it: Iterator[(K, V)]) {
+class RichPairIterator[K : Ordering, V](val it: Iterator[(K, V)]) {
```

Comment:
Actually, `RichOrderedKeyPairIterator` has the same problem. It's an implicitly converted class whose conversion has a not-fully-constrained implicit argument. I'll have to add a method to the `OrderedRDD` object.